src/trustyai/utils/server/server.py

```diff
@@ -0,0 +1,29 @@
+"""
+Server module
+"""
+# pylint: disable = import-error, too-few-public-methods
+from kubernetes import config, client
```

Comment:
We could add here a check to see if the optional `trustyai[api]` dependencies are installed. An example is at https://github.com/trustyai-explainability/trustyai-explainability-python/blob/e40dc588c1264f26beff10ed94024b06864a8e68/src/trustyai/language/detoxify/tmarco.py#L8